### PR TITLE
[doc] Added example to look for a user's processes/connected

### PR DESCRIPTION
**Pull request by gdestuynder**
NOTE @jvehent : This needs a new generation of the html file as well, since as usual - we don't have exactly the same versions/files for the rendered and I don't wanna clobber everything.

(if you have a solution for that.. ;)

### CODE REVIEW

File: doc/cheatsheet.rst [hunk 1]
```diff
@@ -117,6 +117,18 @@ return files that have unknown keys.
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you know the UID of a user, you can check if he has any process running.
+Additionally, this means that you can find out if he's connected as well, with
+the same command.  In this example `1663` is the UID of the user we're looking
+for.
+
+.. code:: bash
+
+        mig file -path /proc/ -maxdepth 2 -name "^status$" -content "^Uid:.*1663"
```
Comment (jvehent) [inline doc/cheatsheet.rst]: Just a nit, but the regex can be better written as `^Uid:\s+(1664)\s+`

**Comment by jvehent** [comment]
I'm at the mercy of rst2html5, and it's non-deterministic way to generate css classes, so for now, generation of html is on me :)

File: doc/cheatsheet.rst (hunk 1 above)
Comment (mvanotti) [inline doc/cheatsheet.rst]: Typo: Find **if** a user

**Comment by gdestuynder** [comment]
Thanks, updated! (note 1663 is my actual uid. i wonder who's the lucky 1664 ;)

**Comment by jvehent** [comment]
Approved!

**Pull request merged**